### DISCUSSION TITLE
Colliding trains

### DISCUSSION THREAD

**Discussion by RockSoftly** [voxeltycoon/issues#958] [UI&UX]
When tracks are too close together, meeting trains get destroyed. This is seriously discouraging to a player like myself. Also a train on one of the side by side tracks will affect the signals of the adjacent track. This is good in the sense that it can prevent the collision but the signal behavior is wrong and difficult for the user to understand. It would seem preferable to prevent the tracks from being built so close together in the first place.

**Comment by andrewpey** [comment]
Same as #775.
